ValveResourceFormat/IO/GltfModelExporter.cs

```diff
@@ -752,6 +751,73 @@ private IEnumerable<(Node Node, List<int> Indices)> CreateBonesRecursive(Bone bo
                 .Append((node, bone.SkinIndices));
         }
 
+        /// <summary>
+        /// Packs metal and rough maps to metallicRoughnessTexture
+        /// </summary>
+        public class MRTextureGenerator : IDisposable
```

Comment:
```suggestion
        public class MetallicRoughnesTextureGenerator : IDisposable
```

Is this class not useful when extracting materials outside of gltf?

Reply:
Not right now.